app/views/admin/users/index.html.haml

```diff
@@ -10,7 +10,7 @@
     %table.table.table-striped.table-bordered.table-hover.datatable
       %thead
         %th ID
-        %th State
+        %th Confirmation State
```

Comment:
The text of the switch checkbox can say whatever we want it to say. As long as it fits nicely it could say `Confirmed` and `Unconfirmed`